openhands/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -24,14 +24,10 @@ RUN apt-get update && \
     apt-get install -y --no-install-recommends \
         wget curl ca-certificates sudo apt-utils git jq tmux build-essential ripgrep ffmpeg \
         coreutils util-linux procps findutils grep sed \
-        {%- if (base_image.endswith(':latest') or base_image.endswith(':24.04') or ('mswebench' in base_image)) -%}
```

Comment:
A pattern used a few times in this PR: Avoid trying to determine the right package name from the image name. Using an OR, just try installing one, fallback on the other. If both fail then we fail.